ElectronClient/gui/MainScreen.jsx

```diff
@@ -242,17 +242,38 @@ class MainScreenComponent extends React.Component {
 				},
 			});
 		} else if (command.name === 'moveToFolder') {
-			const startFolders = [];
+			let folders = await Folder.allAsTree();
+			let startFolders = [];
 			const maxDepth = 15;
 
+			const folderSort = (folders) => {
+				folders.sort((a, b) => {
+					if (a.indentDepth && b.indentDepth) {
+						if (a.indentDepth === b.indentDepth && a.parentId === b.parentId) {
+							// sensitivity accent will treat accented characters as different
+							// but treats caps as equal
+							return a.label.localeCompare(b.label, undefined, { sensitivity: 'accent' });
```

Comment:
Thanks @Rishgod. Although in open source projects, compatibility is a bigger problem and as laurent suggested to add the `sortFolderTree` function to Folder model, so it will be for sure used for future cases too and if you see compatibility of `Intl.Collator`, it's not at all good with Android Webview. 
![image](https://user-images.githubusercontent.com/35633575/79726624-ba7db900-8308-11ea-8439-ca46bb216c80.png)

But for `Sort`, it is quite compatible. So, i'll rather suggest myself to go with this as it will not be a performance issue as much as i know.
![image](https://user-images.githubusercontent.com/35633575/79726691-d1241000-8308-11ea-9a7d-0eac1f704f9f.png)

Reply:
Umm, compatibility should be compared with localCompare not sort because init collator will replace that.

Reply:
Yeah, sorry it just got mixed up. But yeah the compatibility of localeCompare is more. Although thanks for putting out options.